### PR TITLE
Bump to hc 0.0.122

### PR DESCRIPTION
**Pull request by timotree3**
Depends on #12 and #13

### CODE REVIEW

**Comment by timotree3** [comment]
Actually, let's just review and merge this PR (instead of merging the dependencies first), since they have test failures

File: zomes/test/src/lib.rs [hunk 1]
```diff
@@ -16,20 +16,73 @@ fn target() -> ExternResult<EntryHash> {
...
+
+#[derive(Debug, Serialize, Deserialize, SerializedBytes, Clone)]
+pub struct Props {
+    pub skip_proof: bool,
+}
+
+pub fn skip_proof_sb(encoded_props: &SerializedBytes) -> bool {
+    let maybe_props = Props::try_from(encoded_props.to_owned());
+    if let Ok(props) = maybe_props {
+        debug!("Using props : {:?}", props);
```
Comment (robbiecarlton) [inline zomes/test/src/lib.rs:52]: I think we want to remove this log or make it more specific

If we keep it, something like

```suggestion
        debug!("Skip proof check. Props: {:?}", props);
```

File: zomes/test/src/lib.rs [hunk 2]
```diff
@@ -114,21 +176,34 @@ fn validate(data: ValidateData) -> ExternResult<ValidateCallbackResult> {
...
+                            "No Agent Validation Pkg found".to_string(),
+                        ))
+                    }
                 }
-            },
+            }
             None => {
-                return Ok(ValidateCallbackResult::Invalid("Impossible state".to_string()))}
+                return Ok(ValidateCallbackResult::Invalid(
+                    "Impossible state".to_string(),
```
Comment (robbiecarlton) [inline zomes/test/src/lib.rs:204]: Do we want to be more specific here about why this state should be impossible? It's not immediately obvious to me.

```suggestion
                    ""Impossible state (entry being validated will always have a previous header)"".to_string(),
```

**Review by robbiecarlton — APPROVED** [review]
A couple of non blocking suggestions

**Pull request merged**